core/src/main/java/org/elasticsearch/index/indexing/IndexingStats.java

```diff
@@ -155,6 +161,7 @@ public void readFrom(StreamInput in) throws IOException {
             indexCount = in.readVLong();
             indexTimeInMillis = in.readVLong();
             indexCurrent = in.readVLong();
+            indexFailedCount = in.readVLong();
```

Comment:
I imagine we'll need the `if (version.onOrAfter(Version.V_2_1_0))` dance around these reads and writes here. We didn't need it in 2.0 because we don't have multi-version compatibility but this will probably go to 2.1 and that'll have to be compatible with 2.0.